src/utils/firebaseConfig.ts

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore, collection, getDocs, Firestore} from 'firebase/firestore';
+
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web firebaseConfig's Firebase configuration
+const firebaseConfigure = {
```

Comment:
Use a separate .env file to store the API keys and other sensitive data

Reply:
Resolved in the Handling side effects (https://github.com/fcodelabs/training-web/pull/241) PR